### DISCUSSION TITLE
Autocomplete Input(API) doesn't get input value from field

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#13155] [Archive]
### Describe the Bug

https://user-images.githubusercontent.com/9326922/167105043-ebcdecca-c667-4252-a2b0-ad1a0dd17f64.mp4

### To Reproduce

The Autocomplete input (API) interface is not working with dynamic variables. For example, if the URL is /something/?q={{name}} then the URL it is querying is /something/?q=.
So simply put the dynamic variable {{name}} is not working.
Or maybe I am doing it wrong.

### Errors Shown

400 Bad Request

### What version of Directus are you using?

9.10.0

### What version of Node.js are you using?

16

### What database are you using?

MySQL

### What browser are you using?

Chrome, Brave

### How are you deploying Directus?

Digital Ocean Cloud

**Comment by br41nslug** [comment]
This is not currently how the interface works. The URL only accepts one dynamic variable "{{value}}" which is the value of the input and the ResultsPath/TextPath/ValuePath are used to select the correct values returned from the API call.
